### ISSUE TITLE
Migrate to 0.4

### ISSUE THREAD

**Issue by BenGood** [bbodi/notecalc3#52]
The new release 0.4 looks fantastic. What's the best way to bring my existing notes into this version?

**Comment by bbodi** [comment]
Honestly, I haven't even looked at the code myself for at least a year, so I've already forgotten what the new features are. Fortunately, I've kept the changelog updated, so I can advise you to read what has changed under the "Breaking Changes" section. Initially, these are the three items that may affect your existing sheets:
  - Syntax of Line References has been changed to `{3}` instead of `{&[3]}` (the previous syntax caused ambiguity during matrix parsing)
  - `sum` variable is not set to zero anymore at headers. You can do that explicitly by writing `sum = 0`
  - `$` is now a prefix operator, so the expression `12$` is now invalid and should be written as `$12`

**Comment by BenGood** [comment]
yeah that part I can do. But all my notes in by browser local storage don't load at all now that you've deployed 0.4. The old bookmark I used https://bbodi.github.io/notecalc3/notecalc loads a broken page. 

If I change the URL to https://bbodi.github.io/notecalc3/ it loads but doesn't pull in any of my notes.

I tried using devtools to copy the values from local storage to the new URL but that also didn't work

**Comment by bbodi** [comment]
I made accessible the previous version so you can save your data: https://bbodi.github.io/notecalc3/3.0/notecalc

**Comment by BenGood** [comment]
Sweet.
Just one more thing. How do I get multiple tabs in the new version? changelog says I can name tabs (yay) so I guess the feature is still there but I can't find it. All I see is 'home' where I can put one note and then the rest of the guide after that.
thanks!

**Comment by bbodi** [comment]
Yes it is my bad sorry. The feature is there, but since I wanted to release the new version with minimal effort, I decided on the mdbook format as a website, and I didn't plan tabs there. So, unfortunately, for now, there is no way.

Maybe I will put back the old form of the website too with the new version, it's quite easy to do with minimal effort, and then it's a win-win situation (I want to keep the current one, I really like having the calculator and the documentation in one place, and that the documentation has embedded notepads).

**Comment by BenGood** [comment]
For anyone else following along: 
I went to the normal url. It's broken now on V0.4 but don't worry your data is still there. Press F12 to get to developer tools and go to local storage. (in firefox there's a storage tab, then find local storage on the left.) (in chrome it's on the application tab.) 

Copy the value of the notecalc key (starts with "{"tabs":[{"encoded_content")

go to https://bbodi.github.io/notecalc3/3.0/notecalc and go into the dev tools and paste the copied data there. refresh and your data will be there
